scripts/test-ssr/src/commands/main.ts

```diff
@@ -27,6 +28,36 @@ type MainParams = {
   stories: string;
 };
 
+function interceptConsoleLogs() {
+  const originalConsoleError = console.error;
+  const originalConsoleWarn = console.warn;
+
+  console.error = (...args: unknown[]) => {
+    // Ignoring 'aria-description' warning from React 17 as it's a valid prop
+    // https://github.com/facebook/react/issues/21035
+    if (containsAriaDescriptionWarning(args[0] + ' ' + args[1]) && React.version.startsWith('17')) {
+      return;
+    }
+
+    console.log(chalk.bgRed.whiteBright('❌  A console.error() was thrown during rendering HTML:'));
+    originalConsoleError(...args);
+
+    throw new Error('During rendering a console.error() was thrown');
```

Comment:
is this necessary ? as it's kinda dupe of line 42 ?

Reply:
![image](https://github.com/microsoft/fluentui/assets/14183168/0b0c85e3-5ed0-4df5-b2c5-ff2ec28a6100)


To clarify:
- _45_ is needed to throw an actual exception and break the test
- _43_ is needed to log the original output as otherwise it's unclear what happens
- _42_ is needed to make _43_ in the output visible

Does it make sense?

Reply:
yeah I understand the logic, i was rather pointing to the message thrown on line 45. as its not being printed anywhere ?

also the message in your print screen is bit confusing , "error below contains relevant info" -> should be "above" ?

Reply:
> as its not being printed anywhere ?

It's printed below, it didn't fit on the screenshot.

> also the message in your print screen is bit confusing , "error below contains relevant info" -> should be "above" ?

Kinda agree, however there is a stacktrace below